azopsreference/3fc1081d-6105-4e19-b60c-1ec1252cf560/contoso/.AzState/Microsoft.Authorization_policyDefinitions-Deploy-Nsg-FlowLogs.parameters.json

```diff
@@ -37,11 +72,19 @@
                 "roleDefinitionIds": [
                   "/providers/Microsoft.Authorization/roleDefinitions/b24988ac-6180-42a0-ab88-20f7382dd24c"
                 ],
-                "name": "[concat('NetworkWatcher_', field('location'), '/', 'Microsoft.Network', resourceGroup().name, field('name'))]",
+                "name": "[concat('NetworkWatcher_', field('location'), '/', field('name'), '-', resourceGroup().name, '-flowlog' )]",
```

Comment:
"name" can be removed as we don't want policy to check for resource name explicitly (as a general rule). I noticed this is not added by you, but would be grateful if you could help to remove :-)

Reply:
Understood - thanks

Reply:
Without the `name` key the policy does not find any resources in scope. I think this is caused by the flow logs object that we are actually interested in being separate from the NSG itself, in this case it's in the `NetworkWatcherRG`